frame/staking/src/mock.rs

```diff
@@ -550,7 +550,7 @@ impl ExtBuilder {
 		let mut ext = self.build();
 		ext.execute_with(test);
 		ext.execute_with(|| {
-			Staking::do_try_state(System::block_number()).unwrap();
+			Staking::do_try_state(System::block_number(), false).unwrap();
```

Comment:
maybe it would be more clear for the reader to pass one of [`TryStateSelect::Fast`, `TryStateSelect::All`] here instead of bool.